app/packages/core/src/utils/generic.tsx

```diff
@@ -29,7 +29,7 @@ export const prettify = (
     );
   }
 
-  return v;
+  return result;
```

Comment:
ah I see. makes sense! the pretty() utility function is taking care of converting it into string ("True" or "False") in the following code

```
export const prettify = (
  v: boolean | string | null | undefined | number | number[]
): URL | string => {
  if (typeof v === "string") {
    if (isURL(v)) {
      try {
        return new URL(v);
      } catch {}
    }

    return v;
  } else if (typeof v === "number") {
    return Number(v.toFixed(3)).toLocaleString();
  } else if (v === true) {
    return "True";
  } else if (v === false) {
    return "False";```